src/Component/Shop/ProductShop/index.jsx

```diff
@@ -0,0 +1,41 @@
+import ProductItem from "../../ProductItem";
+import "./style.scss";
+import { Container } from "react-bootstrap";
+import { useDispatch, useSelector } from "react-redux";
+import Pagination from "./Pagination";
+import { useEffect } from "react";
+import { getShopProducts } from "../../../store/Slide/ProductsSlide";
+
+function ProductShop() {
+  const dispatch = useDispatch();
+  const shopProductsList = useSelector(
+    (state) => state.products.shop.list.data
+  );
+  const isLoading = useSelector((state) => state.products.shop.isLoading);
+  const filter = useSelector((state) => state.products.shop.filter);
+
+  useEffect(() => {
+    dispatch(getShopProducts(filter));
+  }, [filter, dispatch]);
+
+  const content =
+    shopProductsList &&
+    shopProductsList.map((item) => (
+      <div key={item.id} className="mb-4 col-12 col-sm-6 col-md-4">
+        <ProductItem data={item} />
+      </div>
+    ));
+
+  return (
+    <section className="product_shop">
+      <Container>
+        <div className="product_shop__main">
```

Comment:
```suggestion
    <section className="product-shop">
      <Container>
        <div className="product-main">
```
Nếu nó là 1 cụm từ thì sẽ là product-shop
Trong product-shop có 1 element thì sẽ là product-shop__element (VD: product-shop__toggle, product-shop__button)
Còn nếu không phải là element thì thôi